cli/script/command-executor.ts

```diff
@@ -224,6 +261,62 @@ function appRename(command: cli.IAppRenameCommand): Promise<void> {
         });
 }
 
+function addCollaborator(command: cli.ICollaboratorAddCommand): Promise<void> {
+    throwForInvalidEmail(command.email);
+
+    return getAppId(command.appName)
+        .then((appId: string): Promise<void> => {
+            throwForInvalidAppId(appId, command.appName);
+
+            return sdk.addCollaborator(appId, command.email)
+                .then((): Promise<void> => {
+                    log("Successfully added the \"" + command.email + "\" app as a collaborator to \"" + command.appName +"\".");
+                    var collaboratorListCommand: cli.IDeploymentListCommand = {
+                        type: cli.CommandType.collaboratorList,
+                        appName: command.appName,
+                        displayKeys: false,
+                        format: "table"
+                    };
+
+                    return listCollaborators(collaboratorListCommand);
```

Comment:
Do we need to list the existing collaborators after adding one? It seems like it would be OK if we skipped this to simplify the output of the command.